en/ajax.texy

```diff
@@ -72,22 +72,21 @@ $this->redrawControl(); // invalidates the whole component, every snippet
 echo $this->isControlInvalid('footer'); // -> true
 \--
 
-A component that receives a signal, is automatically invalidated.
+A component which receives a signal is automatically marked for redrawing.
 
-Thanks to invalidation of snippets we know exactly which parts of which elements will it be necessary to re-render.
+Thanks to snippet redrawing we know exactly which parts of which elements should be re-rendered.
 
 Macro `{snippet} … {/snippet}` .{toc: Macro snippet}
 ----------------------------------------------------
 
-Nette is based on the idea of **logical**, not **graphical** elements, i.e. a descendant of Control does not represent a rectangular area on a page, but a logical component that that can render even into various forms. (for instance a hypothetical component DataGrid can have a method for rendering the grid and another one for rendering the “paginator”, and so on). Each element can also be rendered multiple times on a single page; or conditionally, or every time with a different template, etc.
-Therefore it is not possible to call some `render` method of every invalid object. The approach to the render must be the same as when the whole page is being rendered.
+Nette is based on the idea of **logical**, not **graphical** elements, i.e. a descendant of Control does not represent a rectangular area on a page, but a logical component that can be rendered in various forms (for instance a hypothetical component DataGrid can have a method for rendering the grid and another one for rendering the “paginator”, and so on). Each element can also be rendered multiple times on a single page; or conditionally, or every time with a different template, etc. Therefore it is not possible to call some `render` method of every invalid object. The approach to the render must be the same as when the whole page is being rendered.
```

Comment:
já jsem taky poslal vedle druhej PR kterej tenhle odsatavec úplně maže, jen mi to přišlo natolik troufalý, že jsem to radši udělal zvlášť.. :D ale jak jsem to dělal v noci, tak tady mám pár nedodělků stejně ještě..